camerad/archon.cpp

```diff
@@ -1159,7 +1186,7 @@ namespace Archon {
    * This function is overloaded.
    *
    * This version is for future compatibility.
-   * The multiple-controller version will pass a reference to a return string. //TODO
```

Comment:
The **//TODO** is an important tag, understood by Doxygen which makes a nice list of things I need to remember to come back to.